fhir-server/liberty-config/server.env

```diff
@@ -0,0 +1 @@
+TZ=UTC
```

Comment:
probably should have a documentation reference

Reply:
not quite sure what you had in mind on this one.  a comment that points at our user's guide?  or you mean something that points to liberty's use of the TZ env var?

Reply:
points to the use of the TZ env variable. I think a quick statement is probably worthwhile

Reply:
unfortunately I didn't find anything about this in the openliberty docs.  
https://www.ibm.com/support/knowledgecenter/SSEQTP_8.5.5/com.ibm.websphere.base.doc/ae/trun_svr_timezones.html is a reasonable link, but quite old.
https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html is pretty generic, but begs the question "does it work on windows?"

Reply:
I do know it needs to be in the server.env file. We tried -D in the jvm options and that doesn't work.